layout/indicator-data/general/component.tsx

```diff
@@ -61,7 +63,7 @@ type ChartProps = {
 
 const BUTTON_STYLES = 'text-sm mb-2 flex items-center border text-color1 border-gray1 border-opacity-20 py-0.5 px-4 rounded-full mr-4 whitespace-nowrap';
 /** Max bar chart items */
-const MAX_ITEMS = 30;
+const MAX_ITEMS = 35;
```

Comment:
I changed this number because almost all charts have between 30 - 32 items, and they don't need pagination